app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -123,6 +124,38 @@ func (s *ForwardingRuleSyncer) GetLoadBalancerStatus(lbName string) (*status.Loa
 	return status, nil
 }
 
+// ListForwardingRules gets all global forwarding rules and returns info about ones we created.
+func (s *ForwardingRuleSyncer) ListForwardingRules() (string, error) {
+	var rules *compute.ForwardingRuleList
+	var err error
+	if rules, err = s.frp.ListGlobalForwardingRules(); err != nil {
+		fmt.Println("Error getting global forwarding rules:", err)
+		return "", err
+	}
+	// TODO: Should we check for nil rules?
+	glog.V(5).Infof("rules: %+v", rules)
+	if len(rules.Items) == 0 {
+		return "[No loadbalancers found]", nil
+	}
+	var result string = "List of multicluster loadbalancers created:\nName; IP; Clusters\n"
+	for _, item := range rules.Items {
```

Comment:
not in this PR, but eventually we should reuse kubectl printers: https://github.com/kubernetes/kubernetes/blob/44f24d219f9d28121eb1f16863312f790e144882/pkg/printers/printers.go#L123:13